lib/web_ui/test/golden_tests/engine/shadow_golden_test.dart

```diff
@@ -12,7 +12,7 @@ import 'package:web_engine_tester/golden_tester.dart';
 
 import 'scuba.dart';
 
-const Color _kShadowColor = Color.fromARGB(255, 255, 0, 0);
```

Comment:
why the change in test?

Reply:
Making the shadows softer made the red color too subtle in the golden test, so I decided to switch the test to pitch black. Also, black shadows are the most commonly used ones.